ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -765,6 +1128,27 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
         break;
       }
 
+      // Check if op is supported by backend
+      bool is_supported = false;
+
+      // Create nGraph backend
+      BackendManager::GetCurrentlySetBackendName(&ng_backend_type);
+      if (ng_backend_type != "NOP") {
```

Comment:
Need to check if we can avoid backend specific ifs
We may want to modify the unit test. NOP backend indeed does not support "Mul" therefore you are getting `TF Op Mul of type Mul is not supported by backend: NOP`

Reply:
yeah, that is the reason i added condition. @shresthamalik @kanvi-nervana - can we modify this unit test? If so, how? I can make the change if it is minor

Reply:
Also, NOP indeed does not have `is_supported` API